crypto/mem_sec.c

```diff
@@ -191,6 +192,27 @@ void *CRYPTO_secure_zalloc(size_t num, const char *file, int line)
     return CRYPTO_zalloc(num, file, line);
 }
 
+void *CRYPTO_secure_malloc_array(size_t num, size_t size,
```

Comment:
In this file (function `sh_init()` we call `OPENSSL_zalloc()` here:
```
504
505     sh.freelist = OPENSSL_zalloc(sh.freelist_size * sizeof(char *));
506     OPENSSL_assert(sh.freelist != NULL);
507     if (sh.freelist == NULL)
508         goto err;
```
It can get replaced with `OPENSSL_calloc(sh.freelist_size, sizeof (char *))`.